.storybook-s2/docs/Release Notes.mdx

```diff
@@ -4,6 +4,47 @@ export default MDXLayout;
 
 # Release Notes
 
+## v0.7.0
+
+### New Components
+* [TreeView](?path=/docs/treeview--docs)
+
+### Updates
+* [Badge](?path=/docs/badge--docs): Add `overflowMode` prop, fix icon alignment, update typo from `variant="charteuse"` to `variant="chartreuse"`
+* [CardView](?path=/docs/cardview--docs): Fix override stylings on the wrapper when using `renderActionBar`
+* Image: Add `fetchPriority` prop
+* [Menu](?path=/docs/menu--docs): Fix menu item's focus rings from exceeding popover boundaries
+* [Tabs](?path=/docs/tabs--docs): Refactor collapse implementation to fix accessibilty issues
```

Comment:
I think we reverted Tabs collapse entirely before the last release, so this is actually a new feature